kpconv_torch/utils/config.py

```diff
@@ -385,3 +399,29 @@ def save(self):
                 text_file.write(f"epoch_steps = {self.epoch_steps:d}\n")
             text_file.write(f"validation_size = {self.validation_size:d}\n")
             text_file.write(f"checkpoint_gap = {self.checkpoint_gap:d}\n")
+
+    def set_infered_file_path(self, path):
+        self.infered_file = path
+    
+    def get_infered_file_path(self):
+        return self.infered_file
```

Comment:
Getters/setters are not really "pythonic". :-)

You may directly access to the class attribute if needed, that's valid.

Reply:
Removed.